src/librbd/image/OpenRequest.cc

```diff
@@ -277,7 +277,10 @@ Context *OpenRequest<I>::handle_v2_get_immutable_metadata(int *result) {
                << cpp_strerror(*result) << dendl;
     send_close_image(*result);
   } else {
-    send_v2_get_stripe_unit_count();
+    if ((m_image_ctx->features & RBD_FEATURE_STRIPINGV2) != 0)
```

Comment:
Nit: what about ```if (m_image_ctx->features & RBD_FEATURE_STRIPINGV2)``` like https://github.com/ceph/ceph/blob/master/src/tools/rbd/action/Info.cc#L261.

Otherwise, looks good to me.

Reply:
@yangdongsheng :) done.